source/cloud-security/azure/activity-services/services/storage.rst

```diff
@@ -23,7 +23,7 @@ This section explains how to use the Azure portal to archive the Azure activity
 Configuring the Activity log export
 -----------------------------------
 
-1. To export the logs, search for the **Activity log** service. It can be found by typing "Activity" in the search engine. From there, access the **Audit Logs** section and click on **Export Data Settings**.
+1. To export the logs, search for the **Activity log** service. It can be found in **Monitor**. From there, click on **Export Activity Logs**.
```

Comment:
```suggestion
1. To export the logs, search for the **Activity log** service. It can be found in the **Monitor** service menu or by typing "Activity" in the search engine. From there, click on **Export Data Settings**.
```

Reply:
Changed text in commit b926228379179b7d6984f54f8916fabac0b0fc8d.